fetch.bs

```diff
@@ -8713,83 +8713,139 @@ relevant to you. The rest of the parameters are used less frequently, often for
+    "https://stream.example.com", and whose <a for=request>client</a> is <a>this</a>'s
+    <a>relevant settings object</a>.
+
+    <li>
+     <p><a for=/>Fetch</a> <var>request</var>, with
+     <a for=fetch><i>processResponse</i></a> set to the following steps given a
+     <a for=/>response</a> <var>response</var>:
+
+     <ol>
+      <li><p>If <var>response</var> is a <a>network error</a>, then present an error to the user.
+
+      <li><p>Otherwise, if <var>response</var>'s <a for=response>status</a> is not an
+      <a>ok status</a>, present some fallback value to the user.
+
+      <li><p>Otherwise, <a for=ReadableStream>get a reader</a> for <a for=/>response</a>'s
+      <a for=response>body</a>'s <a for=body>stream</a>, and process it using the result of
+      <a>extracting a MIME type</a> from <var>response</var>'s <a for=response>headers list</a>.
+     </ol>
+    </li>
+  </div>
+ </dd>
 
- <dt><a for=fetch><i>useParallelQueue</i></a>
- <dd><p>Takes a <a for=/>boolean</a> that defaults to false. Indicates where the algorithms passed
- as arguments will be invoked. Hopefully most standards will not need this.
+ <dt>Ignore the response
+ <dd>
+  <p>In some cases, there is no need for a <a for=/>response</a> at all, e.g. in the case of
+  <code>navigator.sendBeacon()</code>. Processing a response and passing callbacks to
+  <a for=/>fetch</a> is optional, so omitting the callback would <a for=/>fetch</a> without
+  expecting a response. In such cases, the <a for=/>response</a>'s <a for=response>body</a>'s
+  <a for=body>stream</a> will be discarded, and the caller does not have to worry about downloading
+  the contents unnecessarily.
+
+  <p id=example-no-callback class=example><a for=/>Fetch</a> a <a for=/>request</a> whose
+  <a for=request>URL</a> is "<code>https://fire-and-forget.example.com</code>", <a for=request>method</a> is
+  `<code>POST</code>`, and whose <a for=request>client</a> is <a>this</a>'s
+  <a>relevant settings object</a>.
+ </dd>
 </dl>
 
-<p>When invoked, the <a for=/>fetch</a> operation returns a <a for=/>fetch controller</a>. The
-controller is used for performing actions on a fetch operation that has already started, such as
-<a for="fetch controller" lt=abort>aborting</a> the operation by the user or page logic, or
-<a for="fetch controller" lt=terminate>terminating</a> it due to a browser-internal circumstance.
-
-
+<p>Apart from the callbacks to handle responses, <a for=/>fetch</a> accepts additional callbacks
+for advanced cases.<a for=fetch><i>processEarlyHintsResponse</i></a> is intended specifically for
+<a for=/>responses</a> whose <a for=response>status</a> is 103, and is currently handled only by
+navigations. <a for=fetch><i>processRequestBodyChunkLength</i></a> and
+<a for=fetch><i>processRequestEndOfBody</i></a> notify the client of request body uploading
+progress.
+
+<p>Note that the <a for=/>fetch</a> operation runs <a>in parallel</a>, and it usually posts
```

Comment:
Hmm, can we make this more specific by mentioning that typically the Fetch operation is intended to be called from the main thread[^1], but that its _inner machinations_ break off in parallel at some point? I think we can also say that in this scenario, _by default_ the response is processed _back_ on the main thread via the client's event loop. (I like "by default" instead of "usually," since I think it clarifies the forks a little).

[^1]: I've seen feedback like https://github.com/fedidcg/FedCM/issues/261#issuecomment-1268064407 given at least twice so it might be good to codify this here in the description of how Fetch works with multiple eent loops.

Reply:
Thanks, reworded this a bit.